src/Sdk/DTWebApi/WebApi/RunnerRefreshMessage.cs

```diff
@@ -10,11 +11,40 @@ public sealed class RunnerRefreshMessage
     {
         public static readonly String MessageType = "RunnerRefresh";
 
+        [DataContract]
+        public sealed class BrokerPackageMetadata
+        {
+            [JsonConstructor]
+            internal BrokerPackageMetadata()
+            {
+            }
+
+            [DataMember(Name = "download_url")]
+            public string DownloadUrl
+            {
+                get;
+                set;
+            }
+
+            [DataMember(Name = "sha256_checksum")]
+            public string HashValue
+            {
+                get;
+                set;
+            }
+
+            [DataMember(Name = "os")]
+            public string Platform
+            {
+                get;
+                set;
+            }
+        }
+
         [JsonConstructor]
         internal RunnerRefreshMessage()
         {
         }
-
         public RunnerRefreshMessage(
```

Comment:
do we still need to have these fields (runnerId, targetVersion, timeout) in the new system?

Reply:
just TargetVersion, I deleted the others.